package.json

```diff
@@ -28,9 +29,11 @@
     "husky": "^8.0.0",
     "jest": "28.1.0",
```

Comment:
kindly take this out along with some `jest.config.js` file since we're now using what you added instead of jest

Reply:
sure. thanks